extensions/rich_text_components/Image/directives/oppia-noninteractive-image.directive.ts

```diff
@@ -98,12 +98,16 @@ angular.module('oppia').directive('oppiaNoninteractiveImage', [
             // don't have loading indicator or try again for showing images in
             // this case. So we directly assign the url to the imageUrl.
             try {
+              ctrl.imageUrl === '';
```

Comment:
What does this do? Should it be inside some `if` or assigned to some variable.

Reply:
I've added a code comment above. This is needed as in translation suggestion the translatable html needs images from the GCS storage and the newly added translation images need it to fetch from the local datastore. So I've changed it such that it will look for local storage first if not found then look into gcs storage.